hazelcast/src/test/java/com/hazelcast/map/impl/mapstore/MapStoreWriteBehindTest.java

```diff
@@ -245,7 +245,7 @@ public void run() {
     public void testOneMemberWriteBehindFlush() {
         TestMapStore testMapStore = new TestMapStore(1, 1, 1);
         testMapStore.setLoadAllKeys(false);
-        int writeDelaySeconds = 2;
+        int writeDelaySeconds = 10;
```

Comment:
since this will wait 10 seconds from now on, we can think of putting this test to nightly category.

Reply:
The test doesn't wait 10 secs, the flush will happen at 10 secs intervals.
The 10 secs only help in the case there is a significant delay (jitter) between instructions, and we end up with a test run of more than the specified time. In other words, the test will take as long as it was taking before, in most cases.

Reply:
ok makes sense